test/server/lib/twitter.test.js

```diff
@@ -25,61 +25,60 @@ describe('server/lib/twitter', () => {
 
     it('with no amount spent', () => {
       const tweet = twitter.compileTweet('monthlyStats', data);
-      expect(tweet).to.equal(`In December, 2 backers joined (@bakkenbaeck, @mziehlke) - you are the best! 🙌
-
-We received $1,277 from 82 backers. Our current balance is $1,200.
+      expect(tweet).to
+        .equal(`In December, 2 financial contributors joined (@bakkenbaeck, @mziehlke) - you are the best! 🙌
 
-Top backers: @webflowapp, @dalmaer, @stickermule
+We received $1,277 from 82 financial contributors. Our current balance is $1,200.
 
-Thank you! 🙏`);
```

Comment:
Why remove the `Thank you` part from the test? I feel like it's great to have the full content there

Reply:
@Betree `Thank you` is no longer part of this tweet because `tweet.length > 280 - 24 - thankyou.length` now.

https://github.com/opencollective/opencollective-api/blob/7190454f9b123f86193bdcf8e5739b417e75ca93/server/lib/twitter.js#L162-L172

Reply:
Sorry, I missed your reply there! Thanks for the details